### PR TITLE
Migration tests

### PR DESCRIPTION
**Pull request by dlebauer**
replaces PR #376 

* added query of `mimetype_id` from the new mimetypes table, 
* replaced the use of `formats.mimetype` with `formats.mimetype_id` in the select and insert statements
* created the `bety_migrations_test` database on ebi-forecast (@gsrohde)
* added `tests/migration-test-ebi-forecast.xml` that uses this database

### CODE REVIEW

File: modules/photosynthesis/DESCRIPTION [hunk 1]
```diff
@@ -9,18 +9,16 @@ Description: The Predictive Ecosystem Carbon Analyzer (PEcAn) is a scientific
...
-LazyLoad: yes
-LazyData: FALSE
-Collate:
-    fitA.R
-    Licor.QC.R
-    plots.R
+    
+SystemRequirements: NA
+OS_type: NA
+License: NA
```
Comment (mdietze) [inline modules/photosynthesis/DESCRIPTION]: Is there a reason that the SystemRequirements, License, Copyright, etc were removed for the photosynthesis module?

File: nohup.out [hunk 2]
```diff
@@ -0,0 +1,398 @@
+Old build still running (file running exists). Use --force to build
```
Comment (mdietze) [inline nohup.out]: nohup.out should not be committed. Please remove and then add to your .gitignore

File: pecan/STATUS [hunk 3]
```diff
@@ -0,0 +1 @@
+META	2015-03-13 17:07:21
```
Comment (mdietze) [inline pecan/STATUS]: pecan/STATUS should not be committed. Previous revisions had finally removed the pecan/pecan directory and it doesn't need to be reintroduced. Please remove both file and folder and then add the whole folder to your .gitignore

File: pecan/dbfiles/posterior/532/prior.distns.csv [hunk 4]
```diff
@@ -0,0 +1,20 @@
+"distn","parama","paramb","n"
```
Comment (mdietze) [inline pecan/dbfiles/posterior/532/prior.distns.csv]: There's a long series of posteriors added in pecan/dbfiles/posterior. If these are tests they should be in the correct place. If they are not tests they should be removed

**Comment by mdietze** [comment]
Given that a number of binary files and large text files were added as part of pecan/dbfiles/posterior, I'd suggest that this pull request be closed and reinitiated from a clean branch that won't cause those files to become part of the 'baggage' we drag around even after they're deleted.

**Comment by dlebauer** [comment]
replaced by #467

**Pull request closed without merging**